journals/apps/journals/models.py

```diff
@@ -640,10 +649,10 @@ def site(self):
     def get_journal(self):
         """ Get journal associated with this page """
         # TOOD - store journal in model so we don't have to search parents hierarchy
-        journal_about = self.get_journal_about_page()
+        journal_about = self._calculate_journal_about_page()
```

Comment:
this one should be 
`journal_about = self.journal_about_page`
as we don't want to calculate it

Reply:
Oof. Morning brain. Changed.